src/components/AppStateChangeHandler.tsx

```diff
@@ -1,38 +0,0 @@
-import { useCallback, useEffect, useRef, useState } from 'react';
-import { AppState, AppStateStatus } from 'react-native';
-import { analyticsV2 } from '@/analytics';
-import store from '@/redux/store';
-import { walletConnectLoadState } from '@/redux/walletconnect';
-
-type AppStateChangeHandlerProps = {
-  walletReady: boolean;
-};
-
-export function AppStateChangeHandler({ walletReady }: AppStateChangeHandlerProps) {
-  const [appState, setAppState] = useState(AppState.currentState);
-  const eventSubscription = useRef<ReturnType<typeof AppState.addEventListener> | null>(null);
-
-  const handleAppStateChange = useCallback(
-    (nextAppState: AppStateStatus) => {
-      if (appState === 'background' && nextAppState === 'active') {
```

Comment:
no longer needed as this was for loading WC v1 when coming back from background app state